server/pbench/bin/index-pbench

```diff
@@ -248,14 +248,14 @@ def convert_to_float(d):
 
 class ResultData(object):
     def __init__(self, ptb, experiment, opctx):
-        self.run_metadata = {
-            "runtstamp": ptb.start_run,
-            "runid": ptb.md5sum,
-            "experiment": experiment,
-        }
+        self.run_metadata = _dict_const([
+            ("runtstamp", ptb.start_run),
+            ("runid", ptb.md5sum),
+            ("experiment", experiment)
+        ])
```

Comment:
Is this different from
     _dict_const(runtstamp = ptb.start_run, ...)

? Not sure why you are using a list of tuples here. Is "experiment" causing a problem?

Reply:
That syntax would also work here, but the list syntax is usable in more places, but I did not try to be consistent since both approaches are valid.